src/index.css

```diff
@@ -29,21 +32,93 @@
 }
 
 .active-link {
-  color: blue;
+  color: #121212;
 }
 
 a {
   text-decoration: none;
+  color: grey;
+}
+
+.logo a {
+  color: #0290ff;
+}
+
+a:active {
+  color: #121212;
+}
+
+.form {
+  padding-top: 30px;
+  border-top: 1px solid #e8e8e8;
+}
+
+.add-book-heading {
+  margin-bottom: 30px;
+  color: #888;
 }
 
 .book-div {
-  border: 1px solid grey;
-  box-shadow: -1px 0 5px 0;
+  border-radius: 4px;
+  border: 1px solid rgb(239, 239, 239);
+  background-color: #fff;
   margin-top: 20px;
   margin-bottom: 20px;
   padding: 20px 25px;
 }
 
+.book-title {
+  width: 22.813rem;
+  height: 1.813rem;
+  margin: 0.188rem 9.688rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
```

Comment:
- the design is good, but let's make look great.

- [ ] the remove btn should be colored not having a bg-color.
- [ ] also narrow the width if you will not add chapter and progress to it.
- [ ] lastly, add edits and comment as the design has.